hoptimator-k8s/src/main/java/com/linkedin/hoptimator/k8s/K8sTableTriggerDeployer.java

```diff
@@ -0,0 +1,90 @@
+package com.linkedin.hoptimator.k8s;
+
+import com.linkedin.hoptimator.k8s.models.V1alpha1TableTrigger;
+import com.linkedin.hoptimator.k8s.models.V1alpha1TableTriggerList;
+import com.linkedin.hoptimator.k8s.models.V1alpha1TableTriggerSpec;
+import io.kubernetes.client.openapi.models.V1Container;
+import io.kubernetes.client.openapi.models.V1Job;
+import io.kubernetes.client.openapi.models.V1JobSpec;
+import io.kubernetes.client.openapi.models.V1ObjectMeta;
+import io.kubernetes.client.openapi.models.V1PodSpec;
+import io.kubernetes.client.openapi.models.V1PodTemplateSpec;
+import io.kubernetes.client.util.Yaml;
+
+import java.sql.SQLException;
+import java.util.Collections;
+import java.util.Properties;
+
+class K8sTableTriggerDeployer extends K8sDeployer<V1alpha1TableTrigger, V1alpha1TableTriggerList> {
```

Comment:
It looks like this class doesn't do anything that a `TableTemplate` couldn't do? Can we just use a `TableTemplate`?